docs/knowledgebase/solution-deployment-using-rwx-volume-is-stuck-in-a-container-creating-state.md

```diff
@@ -48,7 +48,7 @@ Events:
 
 1. To get the Community Edition licence, register your cluster through the [Ondat SaaS platform](https://portal.ondat.io/) and generate a licence so that it can be applied to your cluster.
 
-	> 💡 If you already have an Ondat cluster that is connected to the Ondat SaaS Platform, you can apply the licence automatically to the connected cluster by following the steps below;
+ > 💡 If you already have an Ondat cluster that is connected to the Ondat SaaS Platform, you can apply the licence automatically to the connected cluster by following the steps below;
```

Comment:
...to the connected cluster (remove by) following the steps below**:**

Reply:
fixed